### ISSUE TITLE
Token

### ISSUE THREAD

**Issue by ghost** [DiNitride/Discord-Music-Status#57]
Discord no longer allows access to tokens. Program is unusable.

**Comment by DiNitride** [comment]
Not really anything I can do about that then

**Comment by DiNitride** [comment]
I'll have a look if there is an alternative way of getting a token

**Comment by ghost** [comment]
I found an alternate way. You need to monitor the network in the same tab that you would've accessed the "Applications > Local Storage > Etc." Record Network, search /api/v6 go to Application then authentication, it is working

**Comment by DiNitride** [comment]
Neat, cheers for finding that. Will update instructions tomorrow morning

**Comment by tehFoxx0rz** [comment]
Hi, I really like this bot, and I'd managed with finding the token before this change in Discord, but I'm trying to find the token now, and I've tried my best, but I can't find out anything out about what these instructions mean.
I'd be very appreciative if someone could explain this process in more detail.

It doesn't seem as though the instructions on the main page have been updated, either.

Thanks very much.
